diff --git a/Configurations/unix-Makefile.tmpl b/Configurations/unix-Makefile.tmpl
index 9dc8e18d64fe2f8aabb08e3b5138b056c9432904..69ecb5f0784d0ac954234fddc11b0236d1342b60 100644
--- a/Configurations/unix-Makefile.tmpl
+++ b/Configurations/unix-Makefile.tmpl
@@ -632,7 +632,7 @@ install_sw: install_dev install_engines install_modules install_runtime ## Insta
 
 uninstall_sw: uninstall_runtime uninstall_modules uninstall_engines uninstall_dev ## Uninstall the software and libraries
 
-install_docs: install_man_docs install_html_docs ## Install manpages and HTML documentation
+install_docs:
 
 uninstall_docs: uninstall_man_docs uninstall_html_docs ## Uninstall manpages and HTML documentation
 	$(RM) -r $(DESTDIR)$(DOCDIR)
